# HG changeset patch
# User Julien Bect <jbect@users.sourceforge.net>
# Date 1508217208 -7200
#      Tue Oct 17 07:13:28 2017 +0200
# Node ID 1d2e37668981ef9dacf5563642b14c77e1232576
# Parent  3ed869a14b560fb32183f25a267bead5739262b3
pt-jit.cc: Fix visit_*_command methods in jit_convert class

* libinterp/parse-tree/pt-jit.cc: Fix visit_*_command methods in jit_convert
  class.  More precisely, remove visit_global_command and visit_persistent_command,
  and add missing visit_decl_command.

diff -r 3ed869a14b56 -r 1d2e37668981 libinterp/parse-tree/pt-jit.cc
--- a/libinterp/parse-tree/pt-jit.cc	Sun Nov 12 17:04:43 2017 +0100
+++ b/libinterp/parse-tree/pt-jit.cc	Tue Oct 17 07:13:28 2017 +0200
@@ -353,15 +353,9 @@
 }
 
 void
-jit_convert::visit_global_command (tree_global_command&)
+jit_convert::visit_decl_command (tree_decl_command&)
 {
-  throw jit_fail_exception ("No visit_global_command implemenation");
-}
-
-void
-jit_convert::visit_persistent_command (tree_persistent_command&)
-{
-  throw jit_fail_exception ("No visit_persistent_command implementation");
+  throw jit_fail_exception ("No visit_decl_command implementation");
 }
 
 void
